plugins/activity/plugin.go

```diff
@@ -39,15 +39,15 @@ func configure(plugin *node.Plugin) {
 // broadcastActivityBlock broadcasts a sync beacon via communication layer.
 func broadcastActivityBlock() {
 	activityPayload := payload.NewGenericDataPayload([]byte("activity"))
-
-	time.Sleep(deps.BlockIssuer.Estimate())
-
-	block, err := deps.BlockIssuer.CreateBlock(activityPayload, Parameters.ParentsCount)
-	if err != nil {
-		Plugin.LogWarnf("error creating activity block: %s", err)
-		return
+	for {
+		if estimate := deps.BlockIssuer.Estimate(); estimate > 0 {
+			time.Sleep(estimate)
+		} else {
+			break
+		}
 	}
-	err = deps.BlockIssuer.IssueBlockAndAwaitBlockToBeScheduled(block, Parameters.BroadcastInterval)
+
+	block, err := deps.BlockIssuer.IssuePayload(activityPayload, Parameters.ParentsCount)
```

Comment:
Something merged wrong.

```suggestion
	block, err := deps.BlockIssuer.CreateBlock(activityPayload, Parameters.ParentsCount)
	if err != nil {
		Plugin.LogWarnf("error creating activity block: %s", err)
		return
	}
	err = deps.BlockIssuer.IssueBlockAndAwaitBlockToBeScheduled(block, Parameters.BroadcastInterval)
```